pro/src/context/ReimbursementContext/ReimbursementContext.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { createContext, useContext, useState } from 'react'
+
+import {
+  GetOffererNameResponseModel,
+  GetOffererResponseModel,
+} from 'apiClient/v1'
+
+export interface ReimbursementContextValues {
+  offerers: GetOffererNameResponseModel[] | null
+  selectedOfferer: GetOffererResponseModel | null
+  setOfferers: (offerersValues: GetOffererNameResponseModel[]) => void
+  setSelectedOfferer: (offererValue: GetOffererResponseModel | null) => void
+}
+
+export const ReimbursementContext = createContext<ReimbursementContextValues>({
+  offerers: null,
+  selectedOfferer: null,
+  setOfferers: () => {},
+  setSelectedOfferer: () => {},
+})
+
+export const useReimbursementContext = () => {
+  return useContext(ReimbursementContext)
+}
+
+interface ReimbursementContextProviderProps {
+  children: React.ReactNode
+}
+
+export function ReimbursementContextProvider({
+  children,
+}: ReimbursementContextProviderProps) {
```

Comment:
À mon avis, pas besoin du type intermédiaire.
```suggestion
export function ReimbursementContextProvider(children: React.ReactNode) {
```